fixtures/flight/.gitignore

```diff
@@ -10,6 +10,7 @@
 
 # production
 /build
+/dist
```

Comment:
I think this might be because it's default webpack output name and CRA only overrides it for prod builds. So we get the default here.